lib/auth/auth.go

```diff
@@ -2225,6 +2250,49 @@ func (a *Server) GenerateHostCerts(ctx context.Context, req *proto.HostCertsRequ
 	}, nil
 }
 
+func (a *Server) UnstableAssertSystemRole(ctx context.Context, req proto.UnstableSystemRoleAssertion) error {
```

Comment:
Can you add godocs to this? `UnstableAssertSystemRole` sounds scary to someone not familiar with what's going on.